### PR TITLE
feat: implement one-way ANOVA

### PR DESCRIPTION
**Pull request by targos**

### CODE REVIEW

File: src/oneWay.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      groupCentered.reduce((total2, centered) => total2 + centered ** 2, 0),
+    0,
+  );
+  withinGroup /= d2;
+
+  const fValue = betweenGroups / withinGroup;
+  const pValue = cephes.fdtrc(d1, d2, fValue);
+
+  return {
+    rejected: fValue >= alpha,
```
Comment (targos) [inline src/oneWay.ts]:
```suggestion
    rejected: pValue >= alpha,
```

File: src/oneWay.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  const d1 = numClasses - 1;
+  const d2 = data.length - numClasses;
+
+  const allData: number[][] = new Array(numClasses);
+  for (let i = 0; i < numClasses; i++) {
+    allData[i] = [];
+  }
+
+  for (let i = 0; i < data.length; i++) {
+    const index = classToIndex.get(classes[i]) as number;
```
Comment (stropitek) [inline src/oneWay.ts:58]: why do you need as number? Isn't the map supposed to return a number already?

Comment (targos) [reply src/oneWay.ts:58]: it can return `undefined` if the key was not found, but here we are sure it won't be the case so I cast it.

**Review by stropitek — APPROVED** [review]

File: src/oneWay.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      groupCentered.reduce((total2, centered) => total2 + centered ** 2, 0),
+    0,
+  );
+  withinGroup /= d2;
+
+  const fValue = betweenGroups / withinGroup;
+  const pValue = cephes.fdtrc(d1, d2, fValue);
+
+  return {
+    rejected: pValue >= alpha,
```
Comment (targos) [inline src/oneWay.ts]:
```suggestion
    rejected: pValue < alpha,
```

**Pull request merged**